### ISSUE TITLE
Syntax updates

### ISSUE THREAD

**Issue by CajunAvenger** [MagicSetEditorPacks/Full-Magic-Pack#55]
* ~~Add a set field option for changing CARDNAME/LEGENDNAME atom between cardname and "this {maintype}".~~
* ~~Add auto-errata for "precombat main" to "first main"~~
* ~~Add auto-errata for "enters the battlefield" to "enters", may need to wait for Bloomburrow release to see all the variations~~
* ~~need pawprint mana, adding to J, reserving J/[A-Z0-9] for alternate symbols. strangely these don't get the modal linebreaks so we don't need to update modal code again~~
* ~~Gift is gonna need its own script again~~
* ~~Forage keyword can also get procced by "foraging" oops~~
* ~~Nyx is now the built in enchantment frame. this will get applied to mainframe but might be awhile before we can reasonably add it to weirder frames.~~

* Add auto-errata for "postcombat main" to "second main"? will need to see how this one gets errata'd.
* want to add a script for reminder text to check when it should add text like "To forage, {normal reminder text}". probably `action_check(to:"forage", "Exile")`, `action_check(to:"surveil "+param1.value, "Look")`
* might also do something similar for "A Food token is..."
* these also get used when two RTs get fused but that might be out of scope

**Comment by G-e-n-e-v-e-n-s-i-S** [comment]
I added the pawprint symbol, bundled it in this PR -> #53
